crates/op-rpc-types/Cargo.toml

```diff
@@ -10,3 +10,35 @@ homepage.workspace = true
 authors.workspace = true
 repository.workspace = true
 exclude.workspace = true
+
+[dependencies]
+# alloy-rlp = { workspace = true, features = ["arrayvec", "derive"] }
+alloy-primitives = { workspace = true, features = ["rlp", "serde", "std"] }
+# alloy-serde.workspace = true
+# alloy-genesis.workspace = true
+
+# alloy-consensus = { workspace = true, features = ["std", "serde"] }
+# alloy-eips = { workspace = true, features = ["std", "serde"] }
```

Comment:
depending on the situation it may be preferable to access alloy-primitives through the alloy prelude crate with the `rlp` and `serde` flags,`std` is enabled by default

https://github.com/alloy-rs/alloy/blob/main/crates/alloy/src/lib.rs